debian/README.etc

```diff
@@ -1,17 +1,27 @@
-Configuring the Kolibri Debian package:
+# Configuring Kolibri
 
-Kolibri is normally configured in its own environment, for instance in
-its main configuration file `~/.kolibri/options.ini` or through command
-line options such as `kolibri start --port=1234`.
+Kolibri is normally configured using the `options.ini` config file, located
+at `$KOLIBRI_HOME/options.ini`. By default, `$KOLIBRI_HOME` is `~/.kolibri`.
```

Comment:
I'm going to merge this and improve later. I'm not sure that the reader studying `/etc/kolibri/README` will understand neither `KOLIBRI_HOME` nor `~` at this stage.

Am trying to read this as a Debian/Ubuntu user who's just installed Kolibri and found out about the `/etc/kolibri` configuration interface.

Reply:
sounds good, but yeah that's why I linked to https://kolibri.readthedocs.io/en/latest/advanced.html in the line below

Reply:
I hope my subsequent changes are in line with what you were thinking:

Priority 1: Introduce the user to conventional options.ini configuration
Priority 2: Introduce the user to environment-based configuration in the `/etc/kolibri` structure